src/ImageManipulation.php

```diff
@@ -516,7 +516,7 @@ public function PreviewThumbnail()
      */
     public function Thumbnail($width, $height)
     {
-        return $this->Pad($width, $height);
+        return $this->Fill($width, $height);
```

Comment:
`Fill()` keep the aspect ratio but crops the image to specified dimensions but `Fit()` doesn't crop at all. With this test image:

![url](https://cloud.githubusercontent.com/assets/318116/25111467/30a2e74e-243f-11e7-9677-518f8abf971a.jpg)

You can it with `Fit()` and `Fill()`

![fit__](https://cloud.githubusercontent.com/assets/318116/25111538/9423853a-243f-11e7-8198-d36e34bdac11.png)

![silverstripe_-_campaigns](https://cloud.githubusercontent.com/assets/318116/25111539/96aab116-243f-11e7-9e68-c5b5eaf8c035.png)

The circle is still around in `Fill()` screen.